test/src/kbasesearchengine/test/search/ObjectDataTest.java

```diff
@@ -8,6 +8,8 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
This doesn't test any failure conditions

Reply:
what failure conditions?

Reply:
The null and whitespace checks